src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -255,6 +255,10 @@ static ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
     }
 #endif
 
+    if (md_config->devx == UCS_NO) {
```

Comment:
we need to fail also if md_config->devx == UCS_YES, but devx is not found in compile time or runtime

Reply:
I'll add those failures. Bas thing is - failure in open_md will be suppressed by gtest...

Reply:
yeah, but it will be useful when user is running mpi for example